src/app/query-home/utils/brim-query.ts

```diff
@@ -99,16 +99,19 @@ export class BrimQuery implements Query {
   }
 
   toString(): string {
-    let s = (this.current?.pins || [])
-      .filter((p) => !p.disabled)
-      .map<QueryPinInterface>(buildPin)
-      .map((p) => p.toZed())
+    let pinS = []
+    if (!isEmpty(this.current?.pins))
+      pinS = this.current.pins
+        .filter((p) => !p.disabled)
+        .map<QueryPinInterface>(buildPin)
+        .map((p) => p.toZed())
+    let s = pinS
```

Comment:
Did this change anything? Doesn't look like it.

Reply:
It's just a little more careful with how it accesses pins by using `isEmpty()`. I a typeError and this fixed it